docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+
+<dt class="jump-anchor" id="gl-confidential-space">Confidential Space</dt>
+<dd>Confidential Space is a highly secure solution for processing confidential data in Google Cloud Platform, within a trusted execution environment (TEE) in a secure enclave.</dd>
+<dd>For details, see <a href="https://cloud.google.com/docs/security/confidential-space">Confidential Space security overview</a>.</dd>
+
+<dt class="jump-anchor" id="gl-jwt">JSON Web Token (JWT)</dt>
+<dd>A JSON Web Token (JWT) is a type of bearer token, meaning that it contains all the information needed to allow the bearer to get access to a protected resource. With a JWT, the information is structured as a JSON object in a predefined format. It is URL-safe and digitally signed.</dd>
+
+<dt class="jump-anchor" id="gl-oidc-token">OpenID Connect (OIDC) token</dt>
+<dd>OpenID Connect (OIDC) is an identity layer on top of the OAuth 2.0 protocol that allows the client to verify the identity of an end-user based on authentication by an authorization server.</dd>
+<dd>In the context of Confidential Space, the Operator Service is the client and Confidential Space is the authorization server.</dd>
+
+<dt class="jump-anchor" id="gl-tee">TEE</dt>
+<dd>Acronym for Trusted Execution Environment. From the Confidential Space documentation:</dd>
+<dd>"Confidential Space uses a trusted execution environment that is designed to release secrets only to authorized workloads."</dd>
+<dd>For details, see <a href="https://cloud.google.com/docs/security/confidential-space#components-of-a-confidential-space-system">Components of a Confidential Space system</a> (Google Cloud documentation).</dd>
+
+<dt class="jump-anchor" id="gl-workload">Workload</dt>
+<dd>A containerized Docker image that runs in a cloud-based Trusted Execution Environment (see <a href="#gl-tee">TEE</a>) on top of the Confidential Space image.</dd>
+
+</dl>
+
+## Prerequisites
+
+Before setting up your UID2 Operator service in the Google Cloud Platform using Confidential Space, complete these prerequisite steps:
+
+- [Confidential Space Account Setup](#confidential-space-account-setup)
+- [UID2 Operator Account Setup](#uid2-operator-account-setup)
+
+### Confidential Space Account Setup
+
+You can run the UID2 Operator service on any GCP account and project. However, to support attestation, you'll need to create a service account that you can use to run Confidential Space virtual machines (VMs), and then grant specific permissions to the account.
+
+For the account setup, you'll need to provide valid values for the placeholders shown in the following tables.
+
+| Placeholder | Actual Value |
+| :--- | :--- |
+| `{PROJECT_ID}` | The ID of the GCP project that you want the GCP Operator to run in. |
+| `{SERVICE_ACCOUNT_NAME}` | A name you choose for this account. It's best to choose a clear but concise name. (**GWH_YS01 are there any naming constraints? I made that up re naming advice... we can take it out... just a guess.**)  |
```

Comment:
It's GCP service account. It do has constrainst but we don't need to repeat the constratint here. We could provide a link here for reference: https://cloud.google.com/iam/docs/service-accounts-create#creating

> The ID must be between 6 and 30 characters, and can contain lowercase alphanumeric characters and dashes.

Reply:
Updated, thx.